packages/ingest-mongodb-public/src/sources/snooty/snootyAstToMd.ts

```diff
@@ -204,3 +209,59 @@ export const getTitleFromSnootyAst = (node: SnootyNode): string | undefined => {
   ) as SnootyTextNode[];
   return textNodes.map(({ value }) => value).join("");
 };
+
+export const getMetadataFromSnootyAst = (
+  node: SnootyNode
+): Record<string, unknown> => {
+  const facetAndMetaNodes = findAll(
+    node,
+    ({ name }) => name === "facet" || name === "meta"
+  ) as (SnootyFacetNode | SnootyMetaNode)[];
```

Comment:
This works for now. Longer term if we plan to maintain this Snooty AST glue code then it might be nice to move to Zod parsing for these instead of type casts.